awx/main/tasks/system.py

```diff
@@ -786,11 +795,17 @@ def update_inventory_computed_fields(inventory_id):
         return
     i = i[0]
     try:
-        i.update_computed_fields()
+        if i.update_computed_fields() == 0:
+            logger.debug('Duplicate update_inventory_computed_fields task.')
```

Comment:
Per offline conversation, I do not believe that this logging will fire off.

- `Inventory.update_computed_fields` does not seem to return a value (unless I missed it)
- Django's internals raise the "did not affect any rows" error, not any database backend (ref: https://github.com/django/django/blob/eff21d8e7a1cb297aedf1c702668b590a1b618f3/django/db/models/base.py#L1107).  Therefore we'll get a `DatabaseError` anyway, bypassing this line, and that exception *will not* have a `sqlstate` attached.